tensorflow/python/keras/callbacks_test.py

```diff
@@ -1350,6 +1350,33 @@ def target():
       t.join()
       assert not t.is_alive()
 
+  @unittest.skipIf(
+      requests == None,
```

Comment:
@ashahab here is the internal error , can you please take a look 
`Traceback (most recent call last):
 File "<embedded module '_launcher'>", line 165, in run_filename_as_main
 File "<embedded module '_launcher'>", line 39, in _run_code_in_main
 File "/build/work/google3/runfiles/google3/third_party/tensorflow/python/keras/callbacks_test.py", line 26, in <module>
 import requests
ModuleNotFoundError: No module named 'requests'
-- Forge runner: Test failed with exit code 1 `

Reply:
replaced this with a context block after the check